src/test/java/com/kodilla/ecommercee/entities/CartTestSuite.java

```diff
@@ -0,0 +1,142 @@
+package com.kodilla.ecommercee.entities;
+
+import com.kodilla.ecommercee.EcommerceeApplication;
+import com.kodilla.ecommercee.domain.*;
+import com.kodilla.ecommercee.repositories.*;
+import org.junit.Test;
```

Comment:
U mnie dopiero testy się wykonały jak zamieniłem "import org.junit.Test;" na "import org.junit.jupiter.api.Test;", ale to może tylko u mnie, albo może to być kwestia wersji Javy

Reply:
u mnie sytuacja odwrotna. Jak zmieniam na ten import to mi nie przechodzą. Dodam nowego taska, w który postaram sie rozwiazac ten problem z wersjami.

Reply:
Jeszcze zobaczymy jak u innych pójdzie. Może to u mnie jest coś nie tak a innym normalnie zadziała

Reply:
Import powininien być z JUnit 5 (import org.junit.jupiter.api.Test)

Reply:
dzięki za pomoc, zaktualizowane